lib/pages/chat_details/participant_list_item.dart

```diff
@@ -36,17 +38,124 @@ class ParticipantListItem extends StatelessWidget {
       opacity: user.membership == Membership.join ? 1 : 0.5,
       child: ListTile(
         onTap: () async {
-          final result = await showAdaptiveBottomSheet(
-            context: context,
-            builder: (c) => UserBottomSheet(
-              user: user,
-              outerContext: context,
-            ),
-          );
-          if (result is ChatMembersStatus) {
-            if (result == ChatMembersStatus.updated) {
-              onUpdatedMembers?.call();
-            }
+          if (PlatformInfos.isMobile) {
+            await showModalBottomSheet(
+              context: context,
+              shape: const RoundedRectangleBorder(
+                borderRadius: BorderRadius.only(
+                  topLeft: Radius.circular(16),
+                  topRight: Radius.circular(16),
```

Comment:
information about padding, height, width, color, you should move to a style file, so that later we can change easily